bootstrap/bungeecord/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("geyser.platform-conventions")
     id("geyser.modrinth-uploading-conventions")
+    alias(libs.plugins.runwaterfall)
```

Comment:
Waterfall has been deprecated by Paper, and also this probably won't work with the latest direct connection changes. Can we test this or move it to a Bungee run plugin?

Reply:
That doesn't exist. I'll drop this pkugin then.